src/renderer/state/upload/logics.ts

```diff
@@ -197,9 +197,14 @@ const initiateUploadLogic = createLogic({
     let uploads: UploadJob[];
     try {
       uploads = await Promise.all(
-        requests.map((request) =>
-          fms.initiateUpload(request, user, { groupId })
-        )
+          requests.map((request) => {
+                const serviceFields = {
+                  groupId,
+                  multifile: determineIsMultifile(request.file.originalPath),
+                }
+                return fms.initiateUpload(request, user, serviceFields);
```

Comment:
Should `fms.initiateUpload` just handle checking if this is `multifile`?

Reply:
I'll have to look at it again, but ideally yes! I'll do that as part of a separate ticket though.